plugin/src/main/scala/org/ehcache/integrations/play/EhcacheJCacheWrapperModule.scala

```diff
@@ -46,8 +50,24 @@ class EhcacheJCacheWrapperProvider @Inject()(env: Environment, config: Configura
   }
 }
 
+/**
+  *
+  */
 class EhcacheValueWrapper extends ValueWrapper {
   def wrapValue(value: Any, expiration: Duration): Any = {
+    WrappedValueWithExpiry(value, expiration)
```

Comment:
The default expiration is `Duration.Inf`... do we want to avoid wrapping the default to help perf?